src/chains/ethereum/ethereum/tests/provider.test.ts

```diff
@@ -178,49 +178,6 @@ describe("provider", () => {
           await provider.send("debug_traceTransaction", [hash]);
         }, controlEvents);
       });
-      it("emits vm:tx:* events for debug_storageRangeAt", async () => {
```

Comment:
removed as this doesn't send these events any more, since the transaction passed in isn't actually run (and it shouldn't have been run either!)